tests/js/spec/views/settings/organizationApiKeysList.spec.jsx

```diff
@@ -13,34 +15,18 @@ const routes = [
 ];
 
 describe('OrganizationApiKeysList', function () {
-  beforeEach(function () {});
-
-  it('renders', function () {
```

Comment:
I removed this because we already test the workflow in the 2nd test. If rendering throws, then 2nd test will throw anyways